tldextract/tldextract.py

```diff
@@ -250,21 +247,21 @@ def _get_tld_extractor(self):
         return self._extractor
 
     def _get_cached_tld_extractor(self):
-        '''Unpickles the local TLD cache file. Returns None on IOError or other
-        unpickling error, or if this object is not set to use the cache
+        '''Read the local TLD cache file. Returns None on IOError or other
+        error, or if this object is not set to use the cache
         file.'''
         if not self.cache_file:
             return
 
         try:
-            with open(self.cache_file, 'rb') as cache_file:
+            with open(self.cache_file) as cache_file:
                 try:
-                    suffixes = pickle.load(cache_file)
```

Comment:
Ah, you are talking about the comment above, right? It will be solved by `only` coercing when serializing I guess so. Am I missing something?